rfcs/0134-nix-store-layer.md

```diff
@@ -0,0 +1,271 @@
+
+# Alternatives
+[alternatives]: #alternatives
+
+Do programming parts without the documentation and website parts.
+But that feels to me like turning an ongoing shift in focus to a one-off change that is likely to bit-rot.
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+What should the store-only Nix be called?
+
+# Future work
+[future]: #future-work
+
+## Nix Store Team
+
+Now that we have this division in the implementation, we also have the opportunity to leverage it for governance purposes.
+An official, NixOS-foundation-authorized team could be set up to manage store layer design decisions (below the threshold of needing an RFC) that don't affect that the rest of Nix.
+\[Some sort of decision that affect all layers is out of scope, must be deliberated with stakeholders from other layers too, probably should be RFC due to such large scope.\]
+
+To be clear, this is *not* to say we should abandon the idea of Nix as a whole.
+There can still be governance of Nix as a whole; this team, and similar hypothetical, say, Flakes, Nix language, or User Experience teams would ultimately need to report to.
+The goal is not to overreact, but strike a balance between:
+
+1. Making sure Nix as a whole continues to make sense
+2. Make sure layers make sense in isolation not just in the context of the way they are currently used.
+
+## Standardization across projects
+
+If we establish informal interop across store-layer implementations with Guix, a next step would be establish some sort of living standard that both communities have equal say in.
+(Of course, implementations are free to implement features in excess of what the standard requires!)
+
+A new store team, per the above, could lead the process from our end, since the other parts of Nix are not shared with Guix and thus out of scope for this sort of cross-project standardization.
+
+## Stabilization
+
+There is a looming question on how to stabilize Nix's big backlog of unstable features (New CLI, Flakes).
+There is a lot of bad-blood over Flakes, both the feature itself and the way it has been rolled out.
+I think the stabilization process can be an opportunity to heal old wounds.
+
+At a minimum, this can involve stabilizing the new CLI before Flakes.
+But even that that is a lot of new feature surface area to review.
+I think even better is stabilizing just the store-only new CLI first.
+
+This is easily the least controversial part of our unstable feature backlog, and yet there is still plenty to discuss.
+Questions like
+
+- logging
+- store paths on `stdout` at end of build?
+- Should commands like `show-derivation` should use `--json` by default
+- Flat vs hierarchical commands
+- is `--derivation` a good flag? (I think not!)
+
+are all in-scope.
+
+Having a conversation just on this narrow first batch of stabilization both builds trust, and ensures these still-important issues they aren't lost in flame wars over more divisive topics.
```

Comment:
I'm not sure how this part relates to the rest. I suppose the topics listed would be interesting for a potential Nix Store Team to take care of, and maybe you can salvage just that list and an introductory note.
```suggestion
```